service/controller/clusterapi/v29/resource/tccp/template_render_test.go

```diff
@@ -3,31 +3,18 @@ package tccp
 import (
 	"bytes"
 	"context"
-	"encoding/json"
 	"flag"
 	"io/ioutil"
-	"net"
 	"path/filepath"
 	"strconv"
 	"testing"
-	"unicode"
 
-	g8sv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/cluster/v1alpha1"
 	"github.com/giantswarm/micrologger/microloggertest"
 	"github.com/google/go-cmp/cmp"
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
 
-	"github.com/giantswarm/aws-operator/pkg/annotation"
-	"github.com/giantswarm/aws-operator/pkg/label"
-	"github.com/giantswarm/aws-operator/service/controller/clusterapi/v29/controllercontext"
 	"github.com/giantswarm/aws-operator/service/controller/clusterapi/v29/detection"
-)
-
-const (
-	TestClusterID           = "8y5ck"
-	TestMachineDeploymentID = "al9qy"
+	"github.com/giantswarm/aws-operator/service/controller/clusterapi/v29/unittest"
```

Comment:
I refactored the fixture structures here so we can use default data for all the different tests. I am not sure if the package name is fine but I couldn't think of something better now.